runtime/heiko/src/constants.rs

```diff
@@ -72,7 +72,8 @@ pub mod fee {
         type Balance = Balance;
         fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
             // in Kusama, extrinsic base weight (smallest non-zero weight) is mapped to 1/10 CENT:
-            // in Statemine, we map to 1/10 of that, or 1/100 CENT
+            // in Statemine, it's mapped to 1/10 of that, or 1/100 CENT
```

Comment:
Can you elaborate on this `it's mapped to 1/10 of that, or 1/100 CENT`?

Reply:
@kaichaosun the formula will be:

```
coeff_integer * x^(degree) + coeff_frac * x^(degree)
```

`WeightToFeePolynomial` uses it to covert `weight` to balance

and here it's `1/100` CENT because:

```
let q = 100 * Balance::from(ExtrinsicBaseWeight::get());
```
q has been multiplied by 100